src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -412,6 +457,18 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
 
         manager.register_pass<ov::pass::CommonOptimizations>();
 
+        { // To convert to f16 input to boolean which is converted to u8, add abs + ceiling + clamp before convert.
+            type_to_fuse_map type_to_fuse = {{ov::opset10::Convert::get_type_info_static(), fuse_type_to_convert}};
+            precisions_map convert_precision_map = {{ov::element::boolean, ov::element::u8}};
+            manager.register_pass<ov::pass::ConvertPrecision>(convert_precision_map,
+                                                          type_to_fuse,
+                                                          false,
+                                                          false,
+                                                          false);
+
```

Comment:
Can't we merge this change to the convertPrecision in the line452?

Reply:
I tried to merge this change to the line 452. but I think it is hard to merge because precision_map is different.

Reply:
However, I will try it again. thanks.

Reply:
Fixed it.